tests/unit/replicator_tests.py

```diff
@@ -116,6 +118,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @flaky(max_runs=3)
```

Comment:
I'll see if I can get a repro script together to demonstrate this failure to the DBcore team. In the meantime lets rely on flaky to get this passing.